barretenberg/cpp/src/barretenberg/polynomials/polynomial.test.cpp

```diff
@@ -0,0 +1,61 @@
+#include <cstddef>
+#include <gtest/gtest.h>
+
+#include "barretenberg/polynomials/polynomial.hpp"
+
+using namespace bb;
+
+// Simple test/demonstration of shifted functionality
+TEST(Polynomial, Shifted)
+{
+    using FF = bb::fr;
+    using Polynomial = Polynomial<FF>;
+    const size_t SIZE = 10;
```

Comment:
nit: all caps for global `constexpr`s seems more normal to me but not worth another CI run if no other changes